src/modules/contacts/pages/ContactOverviewPage.vue

```diff
@@ -17,34 +17,196 @@
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.newsletter')"
+          :value="contact.profile.newsletterStatus"
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.groups')"
+          :value="contact.profile.newsletterGroups.join(',')"
+        />
+      </AppInfoList>
+    </div>
+
+    <div>
+      <AppHeading>{{ t('contactOverview.contribution') }}</AppHeading>
+      <AppInfoList>
+        <AppInfoListItem
+          :name="t('contacts.data.amount')"
+          :value="
+            contact.contributionAmount
+              ? n(contact.contributionAmount, 'currency')
+              : '-'
+          "
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.period')"
+          :value="contact.contributionPeriod"
+        />
+        <AppInfoListItem
+          v-if="contact.contribution.type === ContributionType.GoCardless"
+          :name="t('contacts.data.payingFee')"
+          :value="contact.contribution.payFee"
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.contributionType')"
+          :value="contact.contribution.type"
+        />
+      </AppInfoList>
+    </div>
+    <div class="row-span-3">
+      <AppHeading>{{ t('contactOverview.about') }}</AppHeading>
+      <p class="mb-5">
+        {{ t('contactOverview.annotation.copy.begin') }}
+        <b>{{ t('contactOverview.annotation.copy.bold') }}</b>
+        {{ t('contactOverview.annotation.copy.end') }}
+      </p>
+
+      <form @submit.prevent="handleFormSubmit">
+        <AppInput
+          v-model="contactAnnotations.notes"
+          input-type="text"
+          :label="'Notes'"
+          class="mb-4"
+        />
+        <AppTextArea
+          v-model="contactAnnotations.description"
+          input-type="text"
+          :label="'Description'"
+        />
+        <AppButton
+          type="submit"
+          variant="link"
+          class="mt-4"
+          :loading="loading"
+          >{{ t('form.saveChanges') }}</AppButton
+        >
+      </form>
+    </div>
+
+    <div>
+      <AppHeading>{{ t('contactOverview.roles') }}</AppHeading>
+      <AppInfoList v-for="role in contact.roles" :key="role.role">
+        <AppInfoListItem :name="t(`common.role.${role.role}`)">
+          <!-- @TODO: make component -->
+          <span class="inline-block w-2 h-2 bg-success rounded-full"></span>
+          {{ formatLocale(role.dateAdded, 'P') }}
+          {{
+            role.dateExpires
+              ? ' → ' + formatLocale(role.dateExpires, 'P')
+              : ' onwards'
+          }}
+        </AppInfoListItem>
+      </AppInfoList>
+    </div>
+    <div>
+      <AppHeading>{{ t('contactOverview.security.title') }}</AppHeading>
+      <p>{{ t('contactOverview.security.whatDoTheButtonsDo') }}</p>
+      <form @submit.prevent="handleSecurityAction">
+        <AppButton
+          type="submit"
+          variant="link"
+          :disabled="!securityButtonsEnabled"
```

Comment:
I'm not a huge fan of inverse logic where possible, could this be more readable as `securityButtonsDisabled` instead?

Reply:
Done: https://github.com/beabee-communityrm/beabee-frontend/pull/104/commits/578f343a4e4679ed35823e36a989aa613b9789b8